packages/sdk/src/funding/index.ts

```diff
@@ -73,7 +72,7 @@ export class FundingSDK {
    * @returns the transaction hash or null if no approval was required
    */
   async approveStateChainGateway(
-    amount: bigint | string | number,
+    amount: bigint,
```

Comment:
changed the type of amounts in the exported interfaces to `bigint` to make this consistent throughout the package